src/graphql/profile/getProfileFollowerOverview.graphql

```diff
@@ -19,4 +19,13 @@ query getProfileFollowerOverview($pubKey: PublicKey!, $limit: Int = 1000, $offse
       ...ConnectionNode
     }
   }
+
+  nftsCreated: nfts(creators: [$pubKey], limit: 1000, offset: 0) {
```

Comment:
Moved the getCollectedBy.graqph query here since it made sense to centralize. We can probably delete the other query file